config/src/config/storage_config.rs

```diff
@@ -26,7 +26,8 @@ impl Default for StorageConfig {
             backup_service_port: 7777,
             dir: PathBuf::from("libradb/db"),
             grpc_max_receive_len: Some(100_000_000),
-            prune_window: None,
+            // At 100 tps on avg, we keep 4~5 days of history.
```

Comment:
nit - add a note to indicate it is for fullnode.

Reply:
this will affect everything though, is this the default behavior that we want for all nodes?